### PR TITLE
COMMERCE-11403 Automate test for COMMERCE-11355 and fix failing tests

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-11403

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 1]
```diff
@@ -490,6 +490,89 @@ definition {
...
+
+			Click(
+				key_entryName = "Minium Portal",
+				key_table = "Channels",
+				locator1 = "CommerceEntry#TABLE_ENTRY_DELETE_BUTTON");
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Then buyer verifies that U-Joint is not available in the Catalog") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:544]:
```suggestion
		task ("Then the buyer asserts that U-Joint is not available in the Catalog") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 2]
```diff
@@ -490,6 +490,89 @@ definition {
...
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			CommerceProducts.searchProduct(search = "U-Joint");
+
+			AssertElementNotPresent(locator1 = "//a[contains(text(),'U-Joint')]");
+		}
+
+		task ("When buyer click on the 'Reorder' button of the placed order") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:557]:
```suggestion
		task ("When the buyer clicks on the 'Reorder' button in the placed order's page") {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**